systems-management/nisysmgmt.yml

```diff
@@ -599,6 +617,8 @@ definitions:
       data:
         description: The actual data.
         type: object
+        additionalProperties:
+          type: object
```

Comment:
Same comment as above: is there any restriction on the type of data that can be in this map that we could document?

Reply:
This is used for grains, packages and feeds, and they are really heterogeneous collections.

Grains use strings, ints, bools, inner dictionaries or lists as elements. Also, grains are evolving as salt evolves, and their policy is - we use a dictionary and everyone is taking whatever it cares about from it, and that way there's no versioning.
Packages have a custom format that's defined by nipkg, which can have an open-ended collection of metadata, and feeds are similar too (they have different format of linux feeds, vs. windows (nipkg) feeds). 

So, there are really no constraints on those.

Reply:
That sounds like great information to include in the description instead of "The actual data." :)

Reply:
I did change the description.